packages/mux-player/package.json

```diff
@@ -8,9 +8,12 @@
   "unpkg": "./dist/mux-player.js",
   "jsdelivr": "./dist/mux-player.js",
   "exports": {
-    "import": "./dist/index.mjs",
-    "require": "./dist/index.cjs.js",
-    "default": "./dist/index.cjs.js"
+    ".": {
```

Comment:
question(blocker): Have you confirmed this doesn't break any of the build tools used by the "out of box" frameworks we target? Minimally, let's make sure all of our workspace example apps still work as expected before merge, and let's have as a required followup task confirming that no web build check builds fail after the canary release.

Reply:
just checked the examples, they all build and the player loads and plays